src/AppBundle/EventListener/RedirectToPreferredLocaleListener.php

```diff
@@ -52,8 +52,8 @@ class RedirectToPreferredLocaleListener
     public function __construct(UrlGeneratorInterface $urlGenerator, $locales, $defaultLocale = null)
     {
         $this->urlGenerator = $urlGenerator;
+        $this->locales = array_filter(explode('|', trim($locales)));
```

Comment:
Well I guess the idea was if someone puts the separator `|` in the beginning or end of the locale list to filter empty strings out?

Then this seems more logical to me:

```php
$this->locales = explode('|', trim($locales, ' |'));